src/sidebar.js

```diff
@@ -349,6 +349,7 @@ module.exports = {
       items: [
         "open-source",
         "open-source/commitment",
+        "ecosystem/community",
```

Comment:
This document seems to be added to categories - Documentation/Open source and Self-hosting. 
When I try to open the document from the Open source cat., the whole site shifts to the Self-hosting section. 
Please anchor this doc in a single category to avoid that ;)

Reply:
I think this document should be in both categories:
- It should be in the open source category, so people can find all the community resources, which are also helpful when using Ory Network.
- On the other hand most of these resources are also very useful when self-hosting Ory, so 

IMO it would not be great to remove it from the self-hosting category. 
But also not great to remove it from the main docs, where most ppl will look. 

If you know a way to determine which way it "shifts", let me know. I think it would be better if it stays in "open source" and shifts there if you open it from the self-hosting docs.

If there are no easy fixes to this (I didnt find any in the docusaurus docs) and it is just a cosmetic issue, I would propose to leave it where it is.

Reply:
Seeing as this is the only blocker, maybe you can take a look at my considerations here and we find a solution to get this merged today @tomekpapiernik .

Reply:
Agreed. I don't have anything at the ready, we'd have to do some research and maybe we'll be able to come up with a solution. Should be okay now, I'll merge this.